src/importexport/musicxml/internal/musicxml/musicxmlpart.h

```diff
@@ -19,33 +19,15 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
+#ifndef MUSICXMLPART_H
+#define MUSICXMLPART_H
 
-#ifndef __IMPORTXMLFIRSTPASS_H__
-#define __IMPORTXMLFIRSTPASS_H__
-
-#include "engraving/types/fraction.h"
-#include "engraving/dom/interval.h"
-#include "musicxmlsupport.h"
+#include "types/fraction.h"
+#include "dom/interval.h"
+#include "musicxmltypes.h"
+#include "musicxmlvoicedesc.h"
 
 namespace mu::engraving {
-typedef std::map<int, VoiceDesc> VoiceList;
-//using Intervals = std::map<Fraction, Interval>;
-
-class MusicXmlIntervalList : public std::map<Fraction, Interval>
-{
-public:
-    MusicXmlIntervalList() {}
-    Interval interval(const Fraction f) const;
-};
-
-class MusicXmlInstrList : public std::map<Fraction, String>
-{
-public:
-    MusicXmlInstrList() {}
-    const String instrument(const Fraction f) const;
-    void setInstrument(const String instr, const Fraction f);
-};
-
 class MusicXmlOctaveShiftList : public std::map<Fraction, int>
```

Comment:
This class could also be moved into the cpp file (header only needs `class MusicXmlOctaveShiftList;`)

(unless I am mistaken and std::vector requires the type to be complete, but I think it didn't)

Reply:
Looks like this should be in the header - the vector needs to know the size of the type.